portfolio/src/main/webapp/script.js

```diff
@@ -16,8 +16,11 @@
  * Adds a random greeting to the page.
  */
 function addRandomGreeting() {
+
+    console.log("It enters the function.");
```

Comment:
Seems like this is for debugging purposes? I'd leave these out of the commits unless you really wish to log this to the console. You can also use console.debug if you really want to keep some debugging messages in.

They're mostly identical in terms of functionality:
https://stackoverflow.com/questions/21876461/difference-between-console-log-and-console-debug

But it makes it more clear to a reader of the code what your intention is here if you use the appropriate console function.